### PR TITLE
0.4.1 migration

### PR DESCRIPTION
**Pull request by poolpitako**

### CODE REVIEW

File: contracts/Strategy.sol [hunk 1]
```diff
@@ -1032,6 +1042,15 @@ contract Strategy is BaseStrategy {
             );
     }
 
+    function ethToWant(uint256 _amtInWei)
+        public
+        view
+        override
+        returns (uint256)
+    {
+        return _amtInWei;
```
Comment (jmonteer) [inline contracts/Strategy.sol:1051]: this should use _fromEth function

File: tests/test_debt_ratio.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from brownie import chain, Wei
 
 
-def test_increase(vault, strategy, gov, wbtc, wbtc_whale, weth, weth_whale, yvETH):
+def xtest_increase(vault, strategy, gov, wbtc, wbtc_whale, weth, weth_whale, yvETH):
```
Comment (jmonteer) [inline tests/test_debt_ratio.py:5]: remove x

**Review by jmonteer — APPROVED** [review]

File: contracts/Strategy.sol (hunk 1 above)
Comment (poolpitako) [reply contracts/Strategy.sol:1051]: can't, not enough space in the strat.

Comment (salazarguille) [reply contracts/Strategy.sol:1051]: Comment:

Based on some recommendations here https://soliditydeveloper.com/max-contract-size (see `medium impact`), we would able to reduce the size using fewer internal functions.

I see some internal functions are called from only one place:
- _takeLendingProfit()
- _claimRewards()
- _depositInYVault()
- _depositToAave(uint256 amount)
- _takeVaultProfit()
- _sellAAVEForWant(uint256 _amount)
- _sellInvestmentForWant(uint256 _amount)
- _buyInvestmentTokenWithWant(uint256 _amount)

Then the changes look good to me.

**Comment by salazarguille** [comment]
Please, review the CI/CD. It is failing.

**Comment by poolpitako** [comment]
I am going to test cloning 0.3.5 strat and add to a 0.4.2 vault before deploying a new version.

File: contracts/Strategy.sol (hunk 1 above)
Comment (poolpitako) [reply contracts/Strategy.sol:1048]: removed `_depositInYVault` and I inlined it for a bit more of space.

**Comment by jmonteer** [comment]
@poolpitako I would rebase master

**Pull request closed without merging**